print.go

```diff
@@ -100,20 +123,31 @@ func (e EditScript) WriteUnified(w io.Writer, ab WriterTo, opts ...WriteOpt) (in
 			seg := e.segs[k]
```

Comment:
GitHub oh so helpfully won't let me put this comment on line 119, where it belongs.

git diff highlighters paints the `@@` lines bold purple. Should we?

Reply:
(Note that it paints the `@@ ... @@` part, but not the context description on the rest of that line. Since we don't have context descriptions (yet), that doesn't matter. But worth noting in a comment if we do go purple.

Reply:
I thought about it. I personally don't care about the color here, and I think it's not nearly as helpful as red/green for the actual diff, so I left it out. I'll add a TODO if anyone wants to tackle it in the future.